packages/insomnia-app/app/common/import.ts

```diff
@@ -171,6 +170,15 @@ export async function importRaw(
     importedDocs[model.type] = [];
   }
 
+  // Add a workspace to the resources if it doesn't exist
+  if (!data.resources.some(resource => resource._type === EXPORT_TYPE_WORKSPACE)) {
+    data.resources.push({
+      ...models.workspace.init(),
+      _id: '__WORKSPACE_ID__',
+      _type: 'workspace',
```

Comment:
```suggestion
      _id: WORKSPACE_ID_KEY,
      _type: EXPORT_TYPE_WORKSPACE,
```

Magic strings are no gooooodddd